src/main/java/teammates/sqllogic/api/SqlEmailGenerator.java

```diff
@@ -1029,6 +1029,18 @@ public EmailWrapper generateNewAccountRequestAcknowledgementEmail(AccountRequest
         return email;
     }
 
+    /**
+     * Generates the email to be sent to instructor when their account request has been rejected by admin.
+     */
+    public EmailWrapper generateAccountRequestRejectionEmail(AccountRequest accountRequest, String title, String content) {
```

Comment:
I think prof mentioned he wanted the email to be BCC-ed to the support email so the admin gets a copy as well